src/domain/RootViewModel.ts

```diff
@@ -117,9 +118,10 @@ export class RootViewModel extends ViewModel {
         }
     }
 
-    _showLogin(loginToken) {
+    _showLogin(loginToken?: string) {
         this._setSection(() => {
             this._loginViewModel = new LoginViewModel(this.childOptions({
+                ...this.options,
```

Comment:
I can't for the life of me figure out why this is required, but typescript complains otherwise. Let me know if anybody else understands it...